packages/attestation-service/migrations/20200924232226-attestation-v4.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+module.exports = {
```

Comment:
do we need another migration in the same release?  why not just add this in the previous migration?

Reply:
because I had it deployed on baklava already